src/web3/web3.service.ts

```diff
@@ -0,0 +1,62 @@
+import { Injectable } from '@nestjs/common';
+import { ConfigService } from '@nestjs/config';
+import * as ContractInstance from '../contracts/Example.json';
+
+// eslint-disable-next-line @typescript-eslint/no-var-requires
+const Web3 = require('web3');
```

Comment:
Could Web3 be imported with import sintax instead of require?

Reply:
I had a lot of problems trying to import the lib using import syntax. I really don't know why I couldn't make it work